python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -783,15 +783,17 @@ def run_coordinator(self) -> "RunCoordinator":
             self._run_coordinator.register_instance(self)
         return self._run_coordinator
 
-    def get_run_queue_config(self) -> Optional["RunQueueConfig"]:
+    def get_concurrency_config(self) -> ConcurrencyConfig:
         from dagster._core.run_coordinator.queued_run_coordinator import QueuedRunCoordinator
 
-        if not isinstance(self.run_coordinator, QueuedRunCoordinator):
-            return None
+        if isinstance(self.run_coordinator, QueuedRunCoordinator):
```

Comment:
eventually we probably want this run queue config to live under concurrency too? but that would be a breaking change i imagine

Reply:
The run queue config does live under concurrency in the new config scheme.  But we still allow you to configure the queued run coordinator (or any run coordinator), so we have to fetch that as well and resolve it.